tests/test_api.py

```diff
@@ -440,6 +444,17 @@ def test_key_class(self):
                 with self.assertRaises(KeyError):
                     Key.from_dict("id", test_key_dict)
 
+        # Test if from_securesystemslib_key removes the private key from keyval
+        # of a securesystemslib key dictionary.
+        sslib_key = generate_ed25519_key()
+        key = Key.from_securesystemslib_key(sslib_key)
+
+        private_key = False
+        for keyval_key in key.keyval.keys():
+            private_key |= keyval_key == 'private'
+        
+        self.assertFalse(private_key)
```

Comment:
```suggestion
        self.assertFalse('private' in key.keyval.keys())
```
or do i miss something?

Reply:
Thaks, @jku! Addressed.